ConsoleRunner.php

```diff
@@ -23,7 +23,8 @@
  * components [
  *     'consoleRunner' => [
  *         'class' => 'vova07\console\ConsoleRunner',
- *         'file' => '@my/path/to/yii' // or an absolute path to console file
+ *         'file' => '@my/path/to/yii', // or an absolute path to console file
+ *         'phpBinaryPath' => '/usr/bin/php'
```

Comment:
I don't like to add hardcoded paths in my config file. This can be very bad experience if your config file is part of the repository and you deploy your code to different servers.

Reply:
It's an example comment, not the actual configuration

Reply:
Anyway php-fpm instance knows nothing about actual php-cli binary, so it should be specified in somehow